Blog Erstellung - IT-Sicherheitskennzeichen (#255)
adessoSE/devblog: _posts/2022-09-02-IT-Sicherheitskennzeichen-.md

---
layout: [post, post-xml]              # Pflichtfeld. Nicht ändern!
title:  "IT-Sicherheitskennzeichen für mehr Sicherheit?"         # Pflichtfeld. Bitte einen Titel für den Blog Post angeben.
date:   2022-09-02 10:25              # Pflichtfeld. Format "YYYY-MM-DD HH:MM". Muss für Veröffentlichung in der Vergangenheit liegen. (Für Preview egal)
modified_date: 2022-09-02             # Optional. Muss angegeben werden, wenn eine bestehende Datei geändert wird.
author_ids: [vschiller]                 # Pflichtfeld. Es muss in der "authors.yml" einen Eintrag mit diesen Namen geben.
categories: [Softwareentwicklung]     # Pflichtfeld. Maximal eine der angegebenen Kategorien verwenden.
tags: [IT-Sicherheit]   # Bitte auf Großschreibung achten.
---

> Hersteller von IT-Produkten können ihre Produkte vom Bundesamt für Sicherheit in der Informationstechnik (BSI) mit dem IT-Sicherheitskennzeichnen auszeichnen lassen. 

Reviewer: ```suggestion
IT-Unternehmen können ihre Produkte vom Bundesamt für Sicherheit in der Informationstechnik (BSI) mit dem IT-Sicherheitskennzeichnen auszeichnen lassen. 
```

> Die Produkthersteller sichern dabei zu, die jeweiligen Sicherheitsrichtlinien ihrer Produktkategorie einzuhalten. 

Reviewer: ```suggestion
Die Unternehmen sichern dabei zu, die jeweiligen Sicherheitsrichtlinien für ihre Produktkategorie einzuhalten. 
```

Das Kennzeichen ist für einen bestimmten Zeitraum gültig. 
Das Siegel wurde am 8. Dezember 2021 eingeführt und wurde auf dem letzten IT-Sicherheitskongress am 1. Februar 2022 erstmalig vergeben. 
Die Grundlage hierfür bietet das IT-Sicherheitsgesetzt 2.0. 

# Wofür ist das IT-Sicherheitskennzeichen gut?

> Das IT-Sicherheitskennzeichen gibt den VerbraucherInnen eine transparente und schnelle Übersicht, ob ein Produkt die von ihnen geforderten Sicherheitsrichtlinien einhält. 

Reviewer: ```suggestion
Das IT-Sicherheitskennzeichen gibt den Nutzenden eine transparente und schnelle Übersicht, ob ein Produkt die von ihnen geforderten Sicherheitsrichtlinien einhält. 
```

> Damit trägt es zu mehr Transparenz im digitalen Verbraucherschutz in Deutschland bei. 

Reviewer: ```suggestion
Damit trägt es zu mehr Transparenz im digitalen Konsumierendenschutz in Deutschland bei. 
```

Author: Ist Verbraucherschutz auch wenn nicht gegendert nicht der Standard? Heißt ja auch noch Verbraucherzentrale & co?

Reviewer: Laut https://geschicktgendern.de/ und https://www.scribbr.de/gendern/woerterbuch/verbraucherschutz-verbraucherinnenschutz/ sollte es gegendert werden. Die Institution heißt zwar Verbraucherzentrale, die Umbenennung von Institutionen aufgrund von gendergerechter Sprache kann aber grade bei deutschen Behörden eine ganze Weile dauern. Ich bin aber auch kein Experte in Sachen gendergerechter Sprache.

> VerbraucherInnen wird es dadurch ermöglicht einfacher und informierter zu beurteilen, ob ein bestimmtes Produkt die Sicherheitsanforderungen einhält. 

Reviewer: ```suggestion
Nutzenden wird es dadurch ermöglicht einfacher und informierter zu beurteilen, ob ein bestimmtes Produkt die Sicherheitsanforderungen einhält. 
```

> Zu jedem IT-Sicherheitskennzeichen gibt es eine Produktseite des BSI auf der aufgelistet ist, ob das Produkt die Sicherheitsanforderungen erfüllt. 

Reviewer: ```suggestion
Zu jedem IT-Sicherheitskennzeichen gibt es eine Produktseite des BSI, auf der aufgelistet ist, ob das Produkt die Sicherheitsanforderungen erfüllt. 
```

Falls es aktuelle Schwachstellen zu dem Produkt gibt, kann dies ebenfalls auf der jeweiligen Produktseite des BSI nachgeschaut werden. 
 
# Der Weg zum IT-Sicherheitskennzeichen 

Nicht alle Produkte können ein IT-Sicherheitskennzeichen beantragen. 
Bislang gibt es die im Bild dargestellten Produktkategorien (Stand 02.09.2022). 

![Produktkategorien IT-Sicherheitskennziechen](/assets/images/posts/ITSicherheitsKennzeichen/Produkt.png)

> Die technischen Richtlinien, die zur jeweiligen Produktkategorie gehören werden ebenfalls an gleicher Stelle zur Verfügung gestellt. Auch wenn es bislang wenige zu sein scheinen, wird die Liste kontinuierlich erweitert werden. 

Reviewer: ```suggestion
Die technischen Richtlinien, die zur jeweiligen Produktkategorie gehören, werden ebenfalls an gleicher Stelle zur Verfügung gestellt. Auch wenn es bislang wenige zu sein scheinen, wird die Liste kontinuierlich erweitert werden. 
```

> Unsere Welt wird immer smarter und vernetzter, dementsprechend werden auch an die Produkte IT Sicherheitsanforderungen gestellt.

Reviewer: ```suggestion
Unsere Welt wird immer smarter und vernetzter, dementsprechend werden auch an die IT-Produkte höhere Sicherheitsanforderungen gestellt.
```

> Zunächst einmal liegt es an den Herstellern, dass das Produkt von Anfang an sicher gestaltet wird (Security by Design) und alle technischen Sicherheitsrichtlinien der jeweiligen Produktgruppe des BSI umgesetzt sind. 

Reviewer: ```suggestion
Zunächst einmal liegt es an den herstellenden Unternehmen, dass das Produkt von Anfang an sicher gestaltet wird (Security by Design) und alle technischen Sicherheitsrichtlinien der jeweiligen Produktgruppe des BSI umgesetzt sind. 
```

> Das IT-Sicherheitskennzeichen kann auf den Seiten des BSI beantragt werden. 

Reviewer: Fehlt hier was? Hier ist ein ziemlich harter Bruch im Text, allerdings geht es im gleichen Absatz weiter..

Author: Ich glaube da war ein Enter zu viel habs korrigiert.

Dafür benötigt das BSI folgende Informationen: 

* Vollständig ausgefüllter Antrag für die jeweilige Produktkategorie
* Produktbilder für die Veröffentlichung auf den Internetseiten des BSI
* Aktueller Auszug aus dem Handelsregister (nicht älter als 6 Monate)
* Formlose Selbsterklärung, dass das Unternehmen sich nicht in Insolvenz oder Liquidation befindet
* Gegebenenfalls weitere ergänzende Anlagen zum Antrag

> Die Formulare befinden sich [hier](https://www.bsi.bund.de/DE/Themen/Unternehmen-und-Organisationen/IT-Sicherheitskennzeichen/fuer-Hersteller/Antrag/IT-SiK-Antrag.html)

Reviewer: ```suggestion
Die Formulare befinden sich [hier](https://www.bsi.bund.de/DE/Themen/Unternehmen-und-Organisationen/IT-Sicherheitskennzeichen/fuer-Hersteller/Antrag/IT-SiK-Antrag.html).
```

> Unter den Anlagen befindet sich auch die Technische Erklärung, die unter anderem folgende Punkte fragt: 

Reviewer: ```suggestion
Unter den Anlagen befindet sich auch die technische Erklärung, die unter anderem folgende Punkte abfragt: 
```
oder ist Technische Erklärung in dem Fall ein feststehender Begriff?

Author: Technische Richtlinie wäre einer, aber für die Erklärung nicht

> -	Wie werden Standardpasswörter falls nötig generiert?

Reviewer: ```suggestion
- Wie werden Standardpasswörter falls nötig generiert?
```

> -	Wie verlaufen die Meldeverfahren zum Umgang mit Schwachstellen?

Reviewer: ```suggestion
- Wie verlaufen die Meldeverfahren zum Umgang mit Schwachstellen?
```

> -	Wie werden zugesicherte Supportzeiten gegenüber den VerbraucherInnen transparent gemacht?

Reviewer: ```suggestion
- Wie werden zugesicherte Supportzeiten gegenüber den Nutzenden transparent gemacht?
```

> -	Wie wird sichergestellt, dass über das Netzwerk keine unautorisierten sicherheitsrelevanten Änderungen an der Konfiguration des Produkts vorgenommen werden können?

Reviewer: ```suggestion
- Wie wird sichergestellt, dass über das Netzwerk keine unautorisierten sicherheitsrelevanten Änderungen an der Konfiguration des Produkts vorgenommen werden können?
```

> -	uvm.

Reviewer: ```suggestion
- uvm.
```

> Auch wird nach dem Handbuch für Benutzende, Richtlinie zum Umgang mit Schwachstellen und falls vorhanden einem technischen Produktblatt gefragt.

Reviewer: Die Richtlinien zum Umgang mit Schwachstellen hattest du oben in den Stichpunkten schon

Author: Habe ich gelöscht

In Summe sind die Anträge und auszufüllenden Dokumente kurzgehalten. 

> Eine technische Prüfung jedes einzelnen Produktes erfolgt nicht, sondern kann stichprobenartig von der Marktaufsicht überprüft werden.

Reviewer: ```suggestion
Eine technische Prüfung jedes einzelnen Produktes erfolgt nicht, stattdessen können einzelne Produkte stichprobenartig von der Marktaufsicht überprüft werden.
```

Falls das IT-Sicherheitskennzeichen vergeben wird, wird es für einen bestimmten Zeitraum vergeben (in der Regel 2 Jahre) und kann anschließend verlängert werden. 

> Zudem kann es von Seiten des BSI widerrufen werden, falls Sicherheitsstandard nicht gehalten werden. 

Reviewer: ```suggestion
Zudem kann es von Seiten des BSI widerrufen werden, falls Sicherheitsstandards nicht eingehalten werden. 
```

> # Welche Informationen können VerbraucherInnen sehen? 

Reviewer: ```suggestion
# Welche Informationen können Nutzende sehen? 
```

Das IT-Sicherheitskennzeichen schafft kein Versprechen, dass das Produkt zu 100% sicher ist. 

> Insbesondere liegt es am Nutzenden Sicherheitsfeatures nicht auszuschalten oder Produkte so zu nutzen, dass der Zugang zu Passwörtern und co. einem Kriminellen erleichtert wird. 

Reviewer: ```suggestion
Insbesondere liegt es am Nutzenden, Sicherheitsfeatures nicht auszuschalten oder Produkte so zu nutzen, dass Dritten der Zugang zu Passwörtern und co. erleichtert wird. 
```

> VerbraucherInnen sind durch das IT-Sicherheitskennzeichen in der Lage wichtige Fakten zu Sicherheitseigenschaften eines vernetzten Produkts auf einen Blick zu erhalten. 

Reviewer: ```suggestion
Nutzende sind durch das IT-Sicherheitskennzeichen in der Lage, auf einen Blick wichtige Fakten zu Sicherheitseigenschaften eines vernetzten Produkts zu erhalten. 
```

> Über einen QR-Code gelangen sie zu der Produktseite des BSI und den aktuellen Status des Produktes. 
> [Hier](https://www.bsi.bund.de/DE/Themen/Unternehmen-und-Organisationen/IT-Sicherheitskennzeichen/Vorschau_Produktseite/Beispiel_Produktinformationsseite.html?nn=936796) eine Beispielseite. 

Reviewer: ```suggestion
Über einen QR-Code gelangen sie zu der Produktseite des BSI und dem aktuellen Status des Produktes, [hier](https://www.bsi.bund.de/DE/Themen/Unternehmen-und-Organisationen/IT-Sicherheitskennzeichen/Vorschau_Produktseite/Beispiel_Produktinformationsseite.html?nn=936796) eine Beispielseite dazu. 
```

> So können sie sehen, dass sich ein Hersteller eines Produkts freiwillig dazu verpflichtet hat die Anforderung des BSI einzuhalten. 

Reviewer: ```suggestion
So können sie sehen, dass sich die Herstellerfirma eines Produkts freiwillig dazu verpflichtet hat, die Anforderung des BSI einzuhalten. 
```

> Dies kann ein Wettbewerbsvorteil darstellen. 

Reviewer: ```suggestion
Dies kann einen Wettbewerbsvorteil darstellen. 
```

Meist ist es nicht auf einen Blick ersichtlich, ob ein Produkt Sicherheitsstandards einhält. 
Unsere Welt wird immer vernetzter, immer mehr Geräte werden smart. 

> Die Analoge Welt wird immer digitaler, für VerbraucherInnen ist es schwierig zu erkennen, ob ein smartes Gerät auch digital sicher ist oder nicht. 

Reviewer: ```suggestion
Die analoge Welt wird immer digitaler, für Nutzende ist es schwierig, zu erkennen, ob ein smartes Gerät auch digital sicher ist oder nicht. 
```

> Wir wissen welchen Stromverbrauch Geräte haben, aber oftmals nicht wie abgesichert unsere Daten sind. 

Reviewer: ```suggestion
Wir wissen, welchen Stromverbrauch Geräte haben, aber oftmals nicht, wie unsere Daten abgesichert sind. 
```

> Dies könnte mehr vertrauen schaffen.

Reviewer: ```suggestion
Das IT-Sicherheitskennzeichen könnte mehr vertrauen schaffen.
```

> Durch das IT-Sicherheitskennzeichen fließt eine weitere wichtige Komponente in die Kaufentscheidung der VerbraucherInnen ein. 

Reviewer: ```suggestion
Das IT-Sicherheitskennzeichen ist eine wichtige Komponente in der Kaufentscheidung von Nutzenden. 
```

> Sie ist eine zusätzliche Komponente die mehr Transparenz in der smarten Welt schafft. 

Reviewer: ```suggestion
Es ist außerdem eine zusätzliche Komponente, die mehr Transparenz in der smarten Welt schafft. 
```

> Es ist nicht das Allheilmittel und garantiert auch nicht, dass ein IT-Prdoukt absolut sicher ist (es können immer neue Angriffsvektoren entstehen), jedoch der erste Schritt für mehr Vertrauen in digitale Produkte und gleichzeitig eine Selbstverpflichtung von Produktherstellern gegenüber ihren KundInnen.

Reviewer: ```suggestion
Es ist nicht das Allheilmittel und garantiert auch nicht, dass ein IT-Prdoukt absolut sicher ist, da immer neue Angriffsvektoren entstehen können.
Es ist jedoch der erste Schritt für mehr Vertrauen in digitale Produkte und gleichzeitig eine Selbstverpflichtung von Produktherstellern gegenüber ihren Kundinnen und Kunden.
```

So können sie sehen, dass sich die Herstellerfirma eines Produkts freiwillig dazu verpflichtet hat, die Anforderung des BSI einzuhalten. 
Dies kann einen Wettbewerbsvorteil darstellen. 
Meist ist es nicht auf einen Blick ersichtlich, ob ein Produkt Sicherheitsstandards einhält. 
Unsere Welt wird immer vernetzter, immer mehr Geräte werden smart. 
Die analoge Welt wird immer digitaler, für Nutzende ist es schwierig, zu erkennen, ob ein smartes Gerät auch digital sicher ist oder nicht. 
Wir wissen, welchen Stromverbrauch Geräte haben, aber oftmals nicht, wie unsere Daten abgesichert sind. 
Das IT-Sicherheitskennzeichen könnte mehr vertrauen schaffen.

Das IT-Sicherheitskennzeichen könnte eine wichtige Komponente in der Kaufentscheidung von Nutzenden werden. 
Es kann außerdem eine zusätzliche Komponente sein, die mehr Transparenz in der smarten Welt schafft. 
Es ist nicht das Allheilmittel und garantiert auch nicht, dass ein IT-Prdoukt absolut sicher ist, da immer neue Angriffsvektoren entstehen können.
Es kann jedoch der erste Schritt für mehr Vertrauen in digitale Produkte und gleichzeitig eine Selbstverpflichtung von Produktherstellern gegenüber ihren Kundinnen und Kunden sein.

> Bislang haben zwei Produktarten das IT-Sicherheitskennzeichen erhalten: E-Mail Dienste und Breindbandrouter. Das IT-Sicherheitskennzeichen ist in der breiten Massen, insbesondere im herstellenden Gewerbe noch nicht bekannt. Dies könnte sich im Laufe der Jahre ändern.

Reviewer: ```suggestion
Bislang haben zwei Produktarten das IT-Sicherheitskennzeichen erhalten: E-Mail-Dienste und Breitbandrouter. Das IT-Sicherheitskennzeichen ist in der breiten Massen, insbesondere im herstellenden Gewerbe, noch nicht bekannt. Dies könnte sich im Laufe der Jahre ändern.
```

Wir wissen, welchen Stromverbrauch Geräte haben, aber oftmals nicht, wie unsere Daten abgesichert sind. 
Das IT-Sicherheitskennzeichen könnte mehr vertrauen schaffen.

> Das IT-Sicherheitskennzeichen ist eine wichtige Komponente in der Kaufentscheidung von Nutzenden. 

Reviewer: Wenn es hierzu noch keine Befragungen o.ä. gibt, solltest du das hier im Konjunktiv formulieren.

Author: Habe ich geändert und noch 2-3 Schlusssätze geändert

Das IT-Sicherheitskennzeichen ist eine wichtige Komponente in der Kaufentscheidung von Nutzenden. 
Es ist außerdem eine zusätzliche Komponente, die mehr Transparenz in der smarten Welt schafft. 
Es ist nicht das Allheilmittel und garantiert auch nicht, dass ein IT-Prdoukt absolut sicher ist, da immer neue Angriffsvektoren entstehen können.

> Es ist jedoch der erste Schritt für mehr Vertrauen in digitale Produkte und gleichzeitig eine Selbstverpflichtung von Produktherstellern gegenüber ihren Kundinnen und Kunden.

Reviewer: Konjunktiv s.o.

Author: Passe ich an
